Client/Ecosia/Extensions/AppSettingsTableViewController+Ecosia.swift

```diff
@@ -17,12 +17,6 @@ extension AppSettingsTableViewController {
             getEcosiaSupportSection(),
             getEcosiaAboutSection()
         ]
-        
-        // Ecosia: Not adding search settings with Bing distribution experiment
-        if !BingDistributionExperiment.isEnabled {
-            sections.insert(getSearchSection(), at: 1)
-        }
```

Comment:
What is the reason to remove this? I guess given `BingDistributionExperiment` still exists it still makes sense to remove Ecosia search settings whenever it's enabled.